concept_website/index.html

```diff
@@ -30,7 +32,35 @@ <h1 id="current_page">Dashboard</h1>
 	  			<a href="settings.html"><div class="menu_button"><i class=material-icons>settings</i></div></a>
 	  		</div>
 	  	</div>
+      <div id="content">
+        <div class="section">
+          <div class="content_module">
+            <div class="module_title"><h2>My Hestia</h2></div>
+          </div>
+          <div class="content_module">
+            <div class="module_title"><h2>Quick Navigation</h2></div>
+          </div>
+        </div>
+      </div>
   	</div>
   </body>
-
+	<!-- Firebase Modules -->
+	<script src="https://www.gstatic.com/firebasejs/4.10.1/firebase-app.js"></script>
+	<script src="https://www.gstatic.com/firebasejs/4.10.1/firebase-auth.js"></script>
+	<script src="https://www.gstatic.com/firebasejs/4.10.1/firebase-database.js"></script>
+	<!-- Firebase Initialization Code -->
+	<script src="https://www.gstatic.com/firebasejs/4.11.0/firebase.js"></script>
+	<script>
+		// Initialize Firebase
+		var config = {
```

Comment:
Code duplication, this code is used 4 times , think about moving it somewhere separate